lib/manager/helm/extract.js

```diff
@@ -0,0 +1,29 @@
+const yaml = require('js-yaml');
+const is = require('@sindresorhus/is');
+
+module.exports = {
+  extractPackageFile,
+};
+
+function extractPackageFile(content) {
+  logger.trace('helm.extractPackageFile()');
+  let deps = [];
+  try {
+    const doc = yaml.safeLoad(content);
+    if (doc && is.array(doc.dependencies)) {
+      deps = doc.dependencies.map(dep => ({
+        depName: dep.name,
+        currentValue: dep.version,
+        helmRepository: dep.repository,
+        datasource: 'helm',
```

Comment:
Is there any possibility of non-helm datasources here? If not then `datasource: 'helm'` could be pushed one level up into the response, rather than be inside each dep.